lib/sisimai/bite/email/opensmtpd.rb

```diff
@@ -0,0 +1,192 @@
+      #   bounce.c/328:
+      #   bounce.c/329:const char *notice_warning2 =
+      #   bounce.c/330:    "    Please note that this is only a temporary failure report.\n"
+      #   bounce.c/331:    "    The message is kept in the queue for up to %s.\n"
+      #   bounce.c/332:    "    You DO NOT NEED to re-send the message to these recipients.\n\n";
+      #   bounce.c/333:
+      #   bounce.c/334:const char *notice_success =
+      #   bounce.c/335:    "    Your message was successfully delivered to these recipients.\n\n";
+      #   bounce.c/336:
+      #   bounce.c/337:const char *notice_relay =
+      #   bounce.c/338:    "    Your message was relayed to these recipients.\n\n";
+      #   bounce.c/339:
+      Re1 = {
+        :begin  => %r/\A[ \t]*This is the MAILER-DAEMON, please DO NOT REPLY to this e[-]?mail[.]\z/,
+        :rfc822 => %r/\A[ \t]*Below is a copy of the original message:\z/,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
+        expired: %r{
+          # smtpd/queue.c:221|  envelope_set_errormsg(&evp, "Envelope expired");
+          Envelope[ ]expired
+        }x,
+        hostunknown: %r{(?:
+          # smtpd/mta.c:976|  relay->failstr = "Invalid domain name";
+           Invalid[ ]domain[ ]name
+          # smtpd/mta.c:980|  relay->failstr = "Domain does not exist";
+          |Domain[ ]does[ ]not[ ]exist
+          )
+        }x,
+        notaccept: %r{
+          # smtp/mta.c:1085|  relay->failstr = "Destination seem to reject all mails";
+          Destination[ ]seem[ ]to[ ]reject[ ]all[ ]mails
+        }x,
+        networkerror: %r{(?>
+          #  smtpd/mta.c:972|  relay->failstr = "Temporary failure in MX lookup";
+           Address[ ]family[ ]mismatch[ ]on[ ]destination[ ]MXs
+          |All[ ]routes[ ]to[ ]destination[ ]blocked
+          |bad[ ]DNS[ ]lookup[ ]error[ ]code
+          |Could[ ]not[ ]retrieve[ ]source[ ]address
+          |Loop[ ]detected
+          |Network[ ]error[ ]on[ ]destination[ ]MXs
+          |No[ ](?>
+             MX[ ]found[ ]for[ ](?:domain|destination)
+            |valid[ ]route[ ]to[ ](?:remote[ ]MX|destination)
+            )
+          |Temporary[ ]failure[ ]in[ ]MX[ ]lookup
+          )
+        }x,
+        securityerror: %r{
+          # smtpd/mta.c:1013|  relay->failstr = "Could not retrieve credentials";
+          Could[ ]not[ ]retrieve[ ]credentials
+        }x,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'OpenSMTPD'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+      def headerlist;  return []; end
+      def pattern;     return Re0; end
+
+      # Parse bounce messages from OpenSMTPD
+      # @param         [Hash] mhead       Message headers of a bounce email
+      # @options mhead [String] from      From header
+      # @options mhead [String] date      Date header
+      # @options mhead [String] subject   Subject header
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->